.github/minimum-reviews.yml

```diff
@@ -0,0 +1,15 @@
+# Number of reviews required to mark the pull request as valid
+reviewsUntilReady: 1
+
+# Number of changes in the pull request to start enforcing the reviewsUntilReady rule
+changesThreshold: 100
```

Comment:
Should we have `reviewsUntilReady` regardless of the changes threshold?
Why do we set it to 100?

Reply:
So I can slowly introduce bitcoin mining code into this repo 99 lines at a time. just kidding. I would never do that. I would make a small PR first that deletes this file then make a large follow up PR that does the bitcoin mining.

Reply:
This got changed to 1